src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
     px: 6,
     py: 3,
   },
   sizes: {
-    sm: {},
-    md: {},
-    lg: {},
+    sm: {
+      height: 11,
+    },
+    md: {
+      height: 12,
+    },
+    lg: {
+      height: 14,
+    },
   },
   variants: {
-    primary: {},
-    secondary: {},
-    tertiary: {},
+    primary: {
+      // token?
+      boxShadow: "0 1.5 1.5 0 rgba(35, 78, 82, 0.08)",
+      _disabled: {
+        backgroundColor: "primary.500",
+        color: "white",
+        // token?
+        opacity: 0.64,
+      },
+      _hover: {
+        backgroundColor: "brand.50",
+        // token?
+        border: "1px solid",
+        borderColor: "brand.800",
+        // token?
+        boxShadow: "0 1 1.5 0 rgba(217, 107, 39, 0.18)",
+        color: "gray.700",
+        fontWeight: "500",
+      },
+      _active: {
+        color: "white",
+        fontWeight: "500",
+        textDecorationLine: "underline",
+      },
+    },
+    secondary: {
+      backgroundColor: "white",
+      // token?
+      border: "2px solid",
+      borderColor: "primary.600",
+      // token?
+      boxShadow: "0 1.5 1.5 0 rgba(44, 122, 123, 0.04)",
+      color: "gray.70",
```

Comment:
Did you mean to do `gray.700` here? This does call attention to a bug where typescript isn't picking up on the theme typings in this file for some reason - I'll look into that.